packages/hierarchies-react/src/test/internal/TreeActions.test.ts

```diff
@@ -390,6 +398,11 @@ describe("TreeActions", () => {
         },
       ]);
 
+      // simulate loader return
+      provider.getNodes.callsFake(() => {
+        return createAsyncIterator([]);
+      });
```

Comment:
maybe this could be done in test case setup to avoid repeating it in many tests?

Reply:
Moved to `beforeEach`

Reply:
please also clean up the commented-out code